src/app/workflow/executions/executions-tab.component.css

```diff
@@ -0,0 +1,48 @@
+/*
+ *    Copyright 2023 OICR. UCSC
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+.metrics-container {
+  display: flex;
```

Comment:
Hmm, I think instead of creating a class, you can achieve similar results using `fxFlex` in the HTML